### PR TITLE
[CI:DOCS] Fix up example description of podman-images.1.md.in

### PR DESCRIPTION
**Pull request by rhatdan**
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Review by flouthoc — APPROVED** [review]
LGTM

File: docs/source/markdown/podman-images.1.md.in [hunk 1]
```diff
@@ -122,6 +122,7 @@ Sort by *created*, *id*, *repository*, *size* or *tag* (default: **created**)
 
 ## EXAMPLE
 
+List all non dangling images in containers/storage:
```
Comment (vrothberg) [inline docs/source/markdown/podman-images.1.md.in:125]: "containers/storage" vs "container storage" vs "local storage" (other man pages)

File: docs/source/markdown/podman-images.1.md.in [hunk 2]
```diff
@@ -167,12 +173,14 @@ ebb91b73692b   <none>                                       <none>
 4526339ae51c   docker.io/library/ubuntu                     latest
 ```
 
+List any image which are not tagged with a name (dangling):
```
Comment (vrothberg) [inline docs/source/markdown/podman-images.1.md.in:176]:
```suggestion
List any image that is not tagged with a name (dangling):
```

File: docs/source/markdown/podman-images.1.md.in [hunk 3]
```diff
@@ -167,12 +173,14 @@ ebb91b73692b   <none>                                       <none>
...
 ```
 # podman images --filter dangling=true
 REPOSITORY   TAG      IMAGE ID       CREATED       SIZE
 <none>       <none>   ebb91b73692b   4 weeks ago   27.2 MB
 ```
 
+List all images in json format:
```
Comment (vrothberg) [inline docs/source/markdown/podman-images.1.md.in:183]:
```suggestion
List all images in JSON format:
```

File: docs/source/markdown/podman-images.1.md.in [hunk 4]
```diff
@@ -122,6 +122,7 @@ Sort by *created*, *id*, *repository*, *size* or *tag* (default: **created**)
 
 ## EXAMPLE
 
+List all non dangling images in local storage:
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-images.1.md.in:125]:
```suggestion
List all non-dangling images in local storage:
```

File: docs/source/markdown/podman-images.1.md.in [hunk 5]
```diff
@@ -216,6 +225,7 @@ registry.access.redhat.com/rhel7            latest   7a840db7f020   2 weeks ago
 registry.fedoraproject.org/fedora           27       801894bc0e43   6 weeks ago          246 MB
 ```
 
+Show difference between listed images in use, versus all images including dangling images:
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-images.1.md.in:228]:
```suggestion
Show the difference between listed images in use versus all images, including dangling images:
```

**Pull request merged**